index.html

```diff
@@ -205,88 +205,16 @@ <h2>Extensibility</h2>
 
 <section><h2>The <code>DOMParser</code> interface</h2>
 
-  <pre class=idl>
-    [Exposed=Window]
-    interface DOMParser {
-      constructor();
-      [NewObject] Document parseFromString(DOMString str, SupportedType type);
-    };
-
-    enum SupportedType {
-      "text/html",
-      "text/xml",
-      "application/xml",
-      "application/xhtml+xml",
-      "image/svg+xml"
-    };
-  </pre>
-
-  <dl class=domintro>
-    <dt><var>domparser</var> = new <a data-link-for=DOMParser>DOMParser</a> ()</var>
-    <dd>Constructs a new DOMParser object.
-
-    <dt><var>document</var> = <var>domparser</var> . <a data-link-for=DOMParser>parseFromString</a> ( <var>str</var>, <var>type</var> )
-    <dd>Parse <var>str</var> using a parser that matches <var>type</var>'s supported MIME types
-    (either XML or HTML), and return a <a>Document</a> object contained the parsed content if
-    successful. If not successful, returns a <a>Document</a> describing the error.
-    <dd>If <var>type</var> does not match a value in the <a>SupportedType</a> enumeration, an
-    exception is thrown [[WEBIDL]].
-  </dl>
+  <span id="dom-domparser"></span>
+  <span id="dom-domparser-parsefromstring"></span>
+  <span id="idl-def-supportedtype"></span>
+  <span id="dom-supportedtype-text/html"></span>
+  <span id="dom-supportedtype-text/xml"></span>
+  <span id="dom-supportedtype-application/xml"></span>
+  <span id="dom-supportedtype-application/xhtml+xml"></span>
+  <span id="dom-supportedtype-image/svg+xml"></span>
```

Comment:
This mechanism for preserving the IDs is clever. We should do the same thing for any other cases where we’re moving something out of a spec, and maybe consider documenting it somewhere as a best practice (though I don’t know often this case comes up). Among other reasons, it ensures any existing links in the Specifications tables in MDN articles still bring readers to the point in the document where they need to be (in this, to the statement that the content’s been moved to the HTML spec).

Reply:
We've been doing it in HTML when moving things around, and Bikeshed has `oldids=""` (https://tabatkins.github.io/bikeshed/#id-gen). There might even be something similar in ReSpec, but I'm not as familiar...